translations/de.json

```
@@ -1,996 +1,1003 @@
+	"Failed to get bootstrapper": "Fehler beim Ermitteln des Bootstrappers",
+	"Failed to get command runner": "Fehler beim Ermitteln des Command Runner",
+	"Failed to get image map": "Fehler beim Ermitteln der Image Map",
+	"Failed to get service URL: {{.error}}": "Fehler beim Ermitteln der Service URL: {{.error}}",
+	"Failed to get temp": "Fehler beim Ermitteln von temp",
 	"Failed to kill mount process: {{.error}}": "Fehler beim Beenden des Bereitstellungsprozesses: {{.error}}",
-	"Failed to list cached images": "",
-	"Failed to list images": "",
-	"Failed to load image": "",
-	"Failed to persist images": "",
-	"Failed to pull image": "",
-	"Failed to pull images": "",
-	"Failed to push images": "",
-	"Failed to read temp": "",
-	"Failed to reload cached images": "",
-	"Failed to remove image": "",
-	"Failed to remove images for profile {{.pName}} {{.error}}": "",
-	"Failed to save config {{.profile}}": "",
-	"Failed to save dir": "",
-	"Failed to save image": "",
-	"Failed to save stdin": "",
-	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY = $ NO_PROXY, {{.ip}}",
-	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}`.": "",
-	"Failed to setup certs": "",
-	"Failed to start container runtime": "",
-	"Failed to start {{.driver}} {{.driver_type}}. Running \"{{.cmd}}\" may fix it: {{.error}}": "",
-	"Failed to stop node {{.name}}": "",
-	"Failed to tag images": "",
-	"Failed to update cluster": "",
-	"Failed to update config": "",
-	"Failed unmount: {{.error}}": "",
-	"Filter to use only VM Drivers": "",
+	"Failed to list cached images": "Auflisten der gecachten Images fehlschlagen",
+	"Failed to list images": "Auflisten der Images fehlgeschlagen",
+	"Failed to load image": "Laden des Images fehlgeschlagen",
+	"Failed to persist images": "Persistierung der Images fehlgeschlagen",
+	"Failed to pull image": "Ziehen des Images fehlgeschlagen",
+	"Failed to pull images": "Ziehen der Images fehlgeschlagen",
+	"Failed to push images": "Remote-Aktualisierung (push) des Images fehlgeschlagen",
+	"Failed to read temp": "Lesen von temp fehlgeschlagen",
+	"Failed to reload cached images": "Erneutes Laden der gecachten Images fehlgeschlagen",
+	"Failed to remove image": "Entfernen des Images fehlgeschlagen",
+	"Failed to remove images for profile {{.pName}} {{.error}}": "Entfernen des Images für Profil {{.pName}} fehlgeschlagen {{.error}}",
+	"Failed to save config {{.profile}}": "Speichern der Konfiguration {{.profile}} fehlgeschlagen",
+	"Failed to save dir": "Speichern des Verzeichnisses fehlgeschlagen",
+	"Failed to save image": "Speichern des Images fehlgeschlagen",
+	"Failed to save stdin": "Speichern der Standard-Eingabe fehlgeschlagen",
+	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY=$NO_PROXY, {{.ip}}",
```

```suggestion
	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,{{.ip}}": "NO_PROXY Env konnte nicht festgelegt werden. Benutzen Sie `export NO_PROXY=$NO_PROXY,{{.ip}}",
```
Need no space after the comma as well